src/components/ebay-menu/README.md

```diff
@@ -0,0 +1,52 @@
+# ebay-menu
+
+## ebay-menu Tag
+
+### ebay-menu Usage
+
+```marko
+<ebay-menu text="text">
+    <ebay-menu-item>item 1</ebay-menu-item>
+    <ebay-menu-item>item 2</ebay-menu-item>
+    <ebay-menu-item>item 3</ebay-menu-item>
+</ebay-menu>
+```
+
+### ebay-menu Attributes
+
+Name | Type | Stateful | Description
+--- | --- | --- | ---
+`type` | String | No | Can be "fake" / "radio" / "checkbox"
+`reverse` | Boolean | No | expand menu flyout to the left
+`fix-width` | Boolean | No | constrain items container width to button width
+`size` | String | No | button size, "small" or "large" (default: medium)
+`priority` | String | No | button priority, "primary" / "secondary" (default) / "none"
+`checked` (radio) | Number | Yes | will set the corresponding index item to `checked` state and use the `aria-checked` attribute in markup
+
+### ebay-menu Events
+
+Event | Data | Description
+--- | --- | ---
+`menu-keydown` | `{ el, index, checked }` |
+`menu-change` (radio) | `{ el, index, checked }` | item changed/checked
+`menu-change` (checkbox) | `{ el, [indexes], [checked] }` | items changed/checked
+`menu-select` (not radio or checkbox) | `{ el, index, checked }` | item clicked
+
+## ebay-menu-item Tag
+
+### ebay-menu-item Usage
+
+```marko
+<ebay-menu-item>item 1</ebay-menu-item>
+```
+
+### ebay-menu-item Attributes
```

Comment:
The new "value" attribute should be documented.

Reply:
Thanks for catching that. I will add it.